.travis/free-check.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+###
+# Checks that "free" is not used unprotected in the main OQS code.
+###
+
+PRINT_GREEN="tput setaf 2"
```

Comment:
I suggest to include .travis/defs.sh here.

Reply:
Done.